applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+
+External dependencies
+*********************
+
+This module utilizes Zephyr :ref:`zephyr:power_management`.
+It depends on :option:`HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`DEVICE_POWER_MANAGEMENT` and :option:`SYS_POWER_DEEP_SLEEP_STATES`.
+
+
+Configuration
+*************
+
+The module is enabled by the ``CONFIG_DESKTOP_POWER_MANAGER_ENABLE`` option.
+
+With ``CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT`` configuration option you can set time after which the application will go to low power mode.
+The ``CONFIG_DESKTOP_POWER_MANAGER_STAY_ON`` let system stay idle also when there are no active connections.
+The ``CONFIG_DESKTOP_POWER_MANAGER_ERROR_TIMEOUT`` set the time after which the device is turned off upon an internal error.
+
+
+Implementation details
+**********************
+
+The :ref:`nrf_desktop_power_manager` is started when the :ref:`nrf_desktop_main` module is ready (reported using ``module_state_event``).
+
+The :ref:`nrf_desktop_power_manager` takes control of the overall system power state.
+Except when entering a deep sleep mode the system will stay in ``SYS_POWER_STATE_ACTIVE`` state.
+When entering the deep sleep mode the module will set system to ``SYS_POWER_STATE_DEEP_SLEEP_1`` state.
+
+
+Power down
+==========
+
+If the device is not connected to the USB, the module counts time since the last user interaction (last HID report sent from the device).
+If the device is connected to the USB, the module remains idle.
+
+On timeout :ref:`nrf_desktop_power_manager` sets the application to the low power state.
+When switching to the low power state :ref:`nrf_desktop_power_manager` will send a `power_down_event``.
+Other application modules react to this event to change their configuration to low power (e.g. turn off LEDs).
+Only after all modules confirmed entering the low power state (by not consuming the ``power_down_event``) the :ref:`nrf_desktop_power_manager` will sets application state to low power.
+
+The next step depends on the BLE connection state.
+ * If the device is connected to the BLE, the module will perform no more actions and the system will remain in an idle state.
+ * If the device is not connected, the module will switch the system into the deep sleep mode.
+   When entering the deep sleep mode, Zephyr suspends all devices and CPU is set to OFF state.
+   To exit deep sleep mode CPU needs an external event.
+
+If disconnection happens while device is in the low power mode, the :ref:`nrf_desktop_power_manager` will set system to the deep sleep mode as described above.
+
+The application can be configured to keep system in the idle state, instead of entering the deep sleep mode, when there are no active connections.
```

Comment:
```suggestion
The application can be configured to keep the system in the idle state when there are no active connections, instead of entering the deep sleep mode.
```